proxy/contracts/schain/TokenManagerLinker.sol

```diff
@@ -90,7 +95,21 @@ contract TokenManagerLinker is AccessControlEnumerableUpgradeable, IMessageRecei
      * 
      * - Function caller has to be granted with {REGISTRAR_ROLE}.
      */
-    function registerTokenManager(TokenManager newTokenManager) external onlyRegistrar {
+    function initialize(MessageProxy newMessageProxyAddress, address linker)
+        external
```

Comment:
Why it is an external?

Reply:
It is not using from insides of the smart contract.